content/docs/contributing.md

```diff
@@ -0,0 +1,138 @@
+# Index
```

Comment:
Would have been better to have this as a separate PR as the discussions on this should not affect the main PR on TL

Reply:
I feel this a lot of text:
- the contributor to read
- for the reviewers to follow through that "the contributor(s)" have read and/or are complying by it

Have a look at mine: https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/CONTRIBUTING.md

Short yet many don't read or try to comply

Reply:
I would split this up into multiple parts, one specific only about the ethos of contribution and rules, etc...

Then what to do technically to start contributing as a separate set of files, you can always back link them to the main README and the CONTRIBUTING pages.

Reply:
ok good idea, I will reshape those documents in a more convenient manner in a separate future PR